src/components/Modal/ModalHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+
+import Heading from '../Heading';
+
+import { MODAL } from '../ThemeIdentifiers';
+
+import * as baseTheme from './Modal.scss';
+
+export type Width = 'small' | 'medium' | 'large' | string;
```

Comment:
@dheeraja00 lets reuse the width type; it's defined at two places within modal.  (but drawer's width is separate from modal's width, keep that one there)